feat: DIA-1325: update response for inference-runs

robot-ci-heartex opened the pull request:
Hi @robot-ci-heartex!

This PR was [created](https://github.com/HumanSignal/label-studio/actions/runs/10500699967) in a response to PRs in upstream repos:
- https://github.com/HumanSignal/label-studio-sdk/pull/294

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10500699967

fern-api[bot] committed:
Merge branch 'develop' into 'fb-dia-1325'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10525072994

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10525072994

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10526565805

fern-api[bot] committed:
Merge branch 'develop' into 'fb-dia-1325'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10528582827

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10528582827

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10528638732

pakelley commented:
/git merge develop
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/10585706739)
> Successfully merged:  291 files changed, 6640 insertions(+), 5672 deletions(-)

pakelley committed:
Merge branch 'develop' into 'fb-dia-1325'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10585706739

pakelley committed:
add has_permissions to ModelRun

fern-api[bot] committed:
Merge branch 'develop' into 'fb-dia-1325'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10604005033

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10604005033

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10605134964

fern-api[bot] committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10608278282

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-dia-1325'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10638454107

robot-ci-heartex committed:
[submodules] Bump HumanSignal/label-studio-sdk version

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/10638454107
